### PR TITLE
Seeding drone [draft]

### PR DESCRIPTION
**Pull request by TilekSydykov**

### CODE REVIEW

**Comment by okalachev** [comment]
At first, see the linter results: https://travis-ci.org/github/CopterExpress/clover/jobs/762820848, https://travis-ci.org/github/CopterExpress/clover/jobs/762820849.

Also, please, don't uses spaces in file names.

File: docs/en/seeding_drone.md [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+Since there are a lot of dangerous and difficult-to-reach landsides for humans to plant, the most viable alternative is to use drones for plantation in those regions.
+
+Seed-firing drones will, as the name suggests, fire seeds into fertile soil to allow millions of trees to grow back after being cut down for industrial use. If the rate of planting exceeds the rate of cutting, eventually we will restore the trees we once felled.
+
+<img src="../assets/seeding_drone/image.jpg" alt="" width="400px"/>
+ 
+## Our Aim
+We will make drones able to hold seeds onboard and drop them in an area which we drove in a special application. We can control the density of the seeds and the height of the drop. We also thought about protection of the seeds from insects, animals and dehydration. We choose the earth ball technique invented by Masanobu Fukuoka, aka Fukuoka Technique. This earth ball contains all needed elements to grow, plant seeds and earth for protection. When we drop it on the ground, the earth ball will hold seeds until it gets the needed amount of water and seeds will begin to grow.
+
+https://www.youtube.com/watch?v=Nz1w59v451U
```
Comment (okalachev) [inline docs/en/seeding_drone.md:19]: What about embedding the video into the article?

Comment (TilekSydykov) [reply docs/en/seeding_drone.md:19]: ok

File: docs/ru/seeding_drone.md [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+
+Как следует из названия, дроны, стреляющие семенами, будут стрелять семенами в плодородную почву, чтобы позволить миллионам деревьев вырасти снова после вырубки для промышленного использования. Если скорость посадки превышает скорость вырубки, в конечном итоге мы восстановим деревья, которые когда-то срубили.
+
+<img src = "../assets/seeding_drone/image.jpg" alt = "" width = "400px" class = "center" />
+
+## Наша цель
+
+Мы сделаем дроны способными удерживать семена на борту и сбрасывать их в область, которую мы проехали в специальном приложении. Мы можем контролировать плотность семян и высоту капли. Мы также думали о защите семян от насекомых, животных и обезвоживания. Мы выбираем технику шара земли, изобретенную Масанобу Фукуока, также известную как Техника Фукуока. Этот земной шар содержит все необходимые элементы для выращивания, семена растений и землю для защиты. Когда мы бросаем его на землю, земной шар будет удерживать семена, пока он не получит необходимое количество воды, и семена не начнут прорастать.
+
+Ссылка на видео Youtube - <https://www.youtube.com/embed/Nz1w59v451U>
```
Comment (okalachev) [inline docs/ru/seeding_drone.md:19]: This would not work, please embed the video with iframe code.

File: docs/ru/seeding_drone.md [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+
+- [Как собрать высевающий механизм на дрон Clover 4.2](#как-собрать-высевной-механизм-на-дрон-clover-42)
+
+- [Как управлять механизмом раздачи](#как-управлять-механизмом-высева)
+
+- [Программирование](#программирование)
+
+## Ссылка на все файлы, используемые в этом проекте
+
+<https://github.com/Sahinysf/TreeSeedQuad>
```
Comment (okalachev) [inline docs/ru/seeding_drone.md:35]: Don't put the links into `<` and `>`.

**Review by okalachev — CHANGES_REQUESTED** [review]

**Comment by okalachev** [comment]
Is this article ready for my final editing and merging?

**Comment by pteacher** [comment]
> 
> 
> Is this article ready for my final editing and merging?

not yet, last fixes

**Comment by okalachev** [comment]
I'm gonna start to edit the article.

File: docs/ru/seeding_drone.md [hunk 4]
```diff
@@ -0,0 +1,329 @@
...
+
+- [Программирование](#программирование)
+
+## Ссылка на все файлы, используемые в этом проекте
+
+[https://github.com/Sahinysf/TreeSeedQuad](https://github.com/Sahinysf/TreeSeedQuad)
+
+СПИСОК ЛИТЕРАТУРЫ
+
+<https://www.nationalgeographic.com/environment/article/deforestation>
```
Comment (okalachev) [inline docs/ru/seeding_drone.md:41]: Is there any names/titles for these links?

File: docs/ru/seeding_drone.md [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+5) Шарики с семенами должны быть идеально круглой формы, иначе они застрянут при броске квадрокоптером.
+6) Вставьте семена (от 1 до 2 семян на семенной шарик для постоянных деревьев, таких как красное дерево, сандал, апельсин, моринга…) (± 5 семян на семенной шарик для овощей, цветов, трав, Клевера…)
+7) Сушите семенные шарики в течение одного-двух дней в затененном месте, если высохнуть должным образом, семенные шарики будут защищены от внешних хищников, таких как птицы, грызуны…
+
+<img src = "../assets/seeding_drone/seedcapsules/3.jpg" alt = "" width = "250px" class = "center" />
+
+Вторая техника - бумажные капсулы с семенами
+
+На этот метод повлияла корейская газета, в которой были семена, которые можно было посадить на улице после прочтения.
+                                                                                                                            Бумажные капсулы с семенами:
```
Comment (okalachev) [inline docs/ru/seeding_drone.md:93]: What are these spaces for?

File: docs/ru/seeding_drone.md [hunk 6]
```diff
@@ -0,0 +1,329 @@
...
+5) Дайте высохнуть на ночь
+
+<img src = "../assets/seeding_drone/seedcapsules/2.jpg" alt = "" width = "400px" class = "center" />
+
+Преимущества бумажных шаров:
+
+- Легко найти материалы
+- экологически чистый
+
+СПИСОК ЛИТЕРАТУРЫ
```
Comment (okalachev) [inline docs/ru/seeding_drone.md:117]: I propose to move all the literature sources lists to the bottom.

File: docs/ru/seeding_drone.md [hunk 7]
```diff
@@ -0,0 +1,329 @@
...
+
+<https://web.archive.org/web/20090115211020/http://www.rmaf.org.ph/Awardees/Biography/BiographyFukuokaMas.htm>
+<http://www.guerrillagarpting.org/ggseedbombs.html>
+
+## Как собрать высевной механизм на дрон Clover 4.2
+
+КАК СБОРКОВАТЬ МЕХАНИЗМ ПОСЕВА
```
Comment (okalachev) [inline docs/ru/seeding_drone.md:124]: Сборковать?

File: docs/ru/seeding_drone.md [hunk 8]
```diff
@@ -0,0 +1,329 @@
...
+
+После завершения шага 4, в разделе Установка защитного кожуха Clover 4.2 в сборе.
```
Comment (okalachev) [inline docs/ru/seeding_drone.md:126]: Какое-то не особо согласованное предложение.

File: docs/ru/seeding_drone.md [hunk 9]
```diff
@@ -0,0 +1,329 @@
...
+    land()
+
+pwm1.stop()
+pwm2.stop()
+GPIO.cleanup()
+```
+
+## Разработано командой MINIONS
+
+### Особая благодарность Международному университету Ала-Тоо за финансирование наборов Clover 4
```
Comment (okalachev) [inline docs/ru/seeding_drone.md]: Wouldn't it be better to put a link here?

**Review by okalachev — CHANGES_REQUESTED** [review]

File: docs/en/seeding_drone.md [hunk 10]
```diff
@@ -0,0 +1,324 @@
...
+
+## Our Aim
+
+We will make drones able to hold seeds onboard and drop them in an area which we drove in a special application. We can control the density of the seeds and the height of the drop. We also thought about protection of the seeds from insects, animals and dehydration. We choose the earth ball technique invented by Masanobu Fukuoka, aka Fukuoka Technique. This earth ball contains all needed elements to grow, plant seeds and earth for protection. When we drop it on the ground, the earth ball will hold seeds until it gets the needed amount of water and seeds will begin to grow.
+
+Youtube video link - [https://www.youtube.com/embed/Nz1w59v451U](https://www.youtube.com/embed/Nz1w59v451U)
+
+We achived to do small seeding missions but we faced some problem about autonomous flying with GPS.
+
+We coated our battery to protect it from cold weather, seeding missions need to start in winter since apple seeds need to stay in a cold place for some time to break dormacy.
```
Comment (okalachev) [inline docs/en/seeding_drone.md:25]: I don't get this phrase. What is dormacy?

File: docs/ru/seeding_drone.md [hunk 11]
```diff
@@ -0,0 +1,328 @@
...
+
+### Пояснение кода для управления серводвигателями
+
+Сервомоторы управляются с помощью сигнала ШИМ (широтно-импульсной модуляции) от Raspberry Pi. ШИМ контролирует количество времени, когда сигнал ВЫСОКИЙ или НИЗКИЙ в течение определенного периода времени. Рабочий цикл - процент времени, когда сигнал ВЫСОКИЙ.
+
+В таблице ниже представлен рабочий цикл серводвигателя SG90 для каждого угла серводвигателя. Чтобы использовать рабочий цикл в коде, нам нужно преобразовать время в проценты, разделив время рабочего цикла на общий период ШИМ.
+
+Что мы получаем:
+
+- Угол поворота -90° или рабочий цикл 2 мс => 1/20 * 100% =  рабочий цикл 5%.
```
Comment (okalachev) [inline docs/ru/seeding_drone.md:184]: Крайне мутные формулировки, может, действительно, оформить в виде таблице, как указано сверху?

**Review by okalachev — CHANGES_REQUESTED** [review]

**Comment by okalachev** [comment]
Please, also review my edits, and say, are they okay for you.

File: docs/en/seeding_drone.md (hunk 10 above)
Comment (Sahinysf) [reply docs/en/seeding_drone.md:25]: It is misspelled, it is actually dormancy

Seed dormancy can be defined as the state or a condition in which seeds are prevented from germinating even under the favourable environmental conditions for germination.

https://byjus.com/biology/seed-dormancy/

File: docs/ru/seeding_drone.md (hunk 4 above)
Comment (Sahinysf) [reply docs/ru/seeding_drone.md]: No, they are just reference websites

File: docs/ru/seeding_drone.md (hunk 3 above)
Comment (Sahinysf) [reply docs/ru/seeding_drone.md]: Ok, we fixed

**Review by TilekSydykov — COMMENTED** [review]
Approve

**Pull request merged**